packages/web/server/queries/numia/earn.ts

```diff
@@ -0,0 +1,122 @@
+import { PricePretty, RatePretty } from "@keplr-wallet/unit";
+import { apiClient } from "@osmosis-labs/utils";
+
+import { Asset } from "~/server/queries/complex/assets";
+
+import { NUMIA_BASE_URL } from ".";
+
+export const EarnStrategyCategories = [
+  "Perps Liquidity",
```

Comment:
NIT: Are these displayed in UI? IMO we should avoid using spaces for keys and we'll use those keys in the localization files

Reply:
Yes they are used in the UI in the "Strategy / Platform" column

Reply:
Also, I want to add that localizing those keys would be a bit of a problem as those are words can be naturally translated but are strictly associated with the lexical context of the crypto world. For instance, removing them would be like removing loan words